packages/stencil/src/components/datacom-breadcrumb/datacom-breadcrumb.tsx

```diff
@@ -0,0 +1,56 @@
+import { Component, Host, h, Prop, Event, EventEmitter, Method } from '@stencil/core';
+import { getSvg } from '../../common/images/icon-provider';
+/**
+ * Breadcrumb element represents  navigation
+ */
+
+@Component({
+  tag: 'datacom-breadcrumb',
+  styleUrl: 'datacom-breadcrumb.css',
+  shadow: true,
+})
+export class DatacomBreadcrumb {
+  @Prop() link?: string;
+  @Prop() text: string;
+  @Prop() index: number;
+  @Prop({ mutable: true }) selected: boolean;
+  @Event() itemClicked: EventEmitter<number>;
+
+  onClickHandler = () => {
```

Comment:
I think you may have overengineered this component. I don't see the need to track state or emit events considering breadcrumbs are really just links? I mean the only complexity I see is adding the chevron icon and handling when the breadcrumb shouldn't be a link as it is the current page (maybe just through a prop). 

Overall, I think you can strip out all the state and event logic from both DatacomBreadcrumb and DatacomBreadcrumbGroup and really simplify this down.

Reply:
I think you can simplify this component further. You don't need this click handler right? I don't see the need for it? Couldn't this just be a prop?

Also I had a comment regarding the selected terminology you used. A breadcrumb isn't 'selected' per se. Is this meant to mean that this link represents the current page?